internal/internal.go

```diff
@@ -14,6 +14,16 @@
 
 package internal
 
+import "time"
+
 // UserAgent is the user agent to be added to the outgoing
 // requests from the exporters.
 const UserAgent = "opencensus-go-v0.1.0"
+
+func MonoticTimeSince(start time.Time) time.Time {
+	// The monotonic clock is used in subtractions hence
```

Comment:
Want to move these comments to the function godoc?

Reply:
Roger that, great idea.